cypress/integration/code-schools.spec.js

```diff
@@ -33,7 +33,16 @@ describe('code schools', function() {
     });
   });
 
-  // /Online Schools test
+  // Online Schools test
+  it('renders only online code schools after clicking on "Online Schools"', () => {
+    cy.contains('Online Schools').click();
+
+    cy.get('[data-testid="SchoolCard"]').each(card => {
+      cy.wrap(card)
+        .get('[data-testid="OnlineSchools"]')
+        .should('has.css', 'color', 'rgba(0, 0, 0, 0.7)');
```

Comment:
Clever! Although, I left a comment on your changes to SchoolCard that should give you a more powerful (less prone to breaking) and declarative (reads how it works) selector.

Using it could look like:
```javascript
cy.wrap(card)
    .get('[data-testid="School has online"]')
    .should('exist');
```